cura/API/Account.py

```diff
@@ -96,6 +96,18 @@ def login(self) -> None:
             return
         self._authorization_service.startAuthorizationFlow()
 
+    @pyqtSlot()
+    def loginWithForcedLogout(self) -> None:
```

Comment:
maybe add a parameter forceLogout to the login function instead?

Reply:
Fixed with https://github.com/Ultimaker/Cura/pull/7743/commits/eac4d3e463c659f9f18ca3a3d7aff603f2561933. Tests have also been adjusted accordingly.